ib-cli/installationiso/data/install-rhcos-and-restore-seed.sh

```diff
@@ -6,8 +6,8 @@ seed_image=${1:-$SEED_IMAGE}
 authfile=${AUTH_FILE:-"/var/tmp/backup-secret.json"}
 ibi_config=${IBI_CONFIGURATION_FILE:-"/var/tmp/ibi-configuration.json"}
 
-# Copy the lca-cli binary to the host
-podman create --authfile "${authfile}" --name lca-cli "${seed_image}" lca-cli
+# Copy the lca-cli binary to the host, pulling it seed image can sometimes fail
+(while ! podman create --authfile "${authfile}" --name lca-cli "${seed_image}" lca-cli ; do exit;sleep 10;done)
```

Comment:
Why is this in a subprocess? Also, the `exit;sleep 10` syntax seems odd here. Won't this just exit from the subprocess when it fails?
```suggestion
while ! podman create --authfile "${authfile}" --name lca-cli "${seed_image}" lca-cli ; do sleep 10;done
```